reference/5-api-publisher/CloudWatch-configuration.md

```diff
@@ -0,0 +1,21 @@
+# Configuration CloudWatch
+
+Enables management of individual connection settings with encryption support for securely storing keys and secrets in the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html).
+
+## Configure AWS SDK
+
+Ensure that the AWS SDK has the necessary [configured AWS credentials](https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/net-dg-config-creds.html).
+
+## Configure API Publisher
+
+Configure the AWS log storage parameters in the configuration file.
+
+![](img/docker/logging-cloudwatch-configuration.png)
+
+Export AWS credentials to consume AWS parameters store inside the container
+
+![](img/docker/setup-aws-credentials.png)
+
+## Looking for a different output format?
```

Comment:
Please add an extra line break above. All headings should be followed by an empty line.

Reply:
fixed